### PR TITLE
Controller

### PR DESCRIPTION
**Pull request by Odongo006**
## Hello
I hope this message finds you well. I would like to submit a pull request for the completion of the project, fulfilling all the specified project requirements. I have implemented the necessary components as per the wireframes provided.

To showcase my understanding of setting up controllers without using the rails generate scaffold command, I have carefully followed the project requirements and made the following additions:

1.     Routes: I have created routes to handle all the required URLs mentioned in the wireframes.
2.     Controller Actions: For each URL, I have implemented the corresponding actions in the relevant controller, ensuring appropriate handling of the data and rendering of views.
3.     View Files: I have created view files for each URL, adhering to the wireframes. The view files contain placeholders written in plain HTML, providing the required structure for the pages.

Please note that I have focused solely on the four required URLs and have not deviated from the project scope.

To verify the setup, I have run the rails routes command in the console, and the output matches the expected routes mentioned in the wireframes.

I would greatly appreciate it if you could take the time to review my code changes, ensuring the correctness and completeness of the implementation. Additionally, any feedback or suggestions for improvement would be invaluable to me.

Thank you for your time and consideration. I look forward to your feedback and guidance.

Best regards,

### CODE REVIEW

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class PostsContoller < ApplicationController
```
Comment (zieeco) [inline app/controllers/posts_controller.rb]: - Hey, we've made a typo here as it should be `PostsController`. Please, kindly update this to fix the error in the routes.

File: app/views/posts/index.html.erb [hunk 2]
```diff
@@ -0,0 +1 @@
+<h1>Listing User Posts</h1>
```
Comment (zieeco) [inline app/views/posts/index.html.erb:1]: `app/views/posts/show.html.erb`

- Let's also display a placeholder text for the post-show page.

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @Odongo006,





Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- Nice controllers. ✔️
- Nice routes. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._


## Optional suggestions


- N/A
 
**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @Odongo006 
Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## Highlights
- Controllers and Routes setup correctly 👏

 ## Required Changes ♻️

- [ ] Kindly add a placeholder text in your [app/views/posts/show.html.erb](https://github.com/Odongo006/Rails_blog_App/pull/5/files#diff-dbe3a3b30b290647eb1f4c401b675c6c0e2f130fac1cadd25c1c447417a6b94b) it is still empty, this was previously requested by the last reviewer 👍🏿 

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by julie-ify — APPROVED** [review]
Hi @Odongo006,

## Highlights!
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Status: Approved ✔️ 

Cheers and Happy coding!👏👏👏

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**